eng/common/TestResources/New-TestResources.ps1

```diff
@@ -63,6 +63,10 @@ param (
     [ValidateSet('AzureCloud', 'AzureUSGovernment', 'AzureChinaCloud', 'Dogfood')]
     [string] $Environment = 'AzureCloud',
 
+    [Parameter()]
```

Comment:
I will update the docs in a separate PR.  I want to unblock the language repo perf test pipelines.  And updating the docs is a little tricky since I'm trying to decide if I want to replace `test-resources.json` with `test-resources.json or perf-resources.json`, or something like `$ResourceType-resources.json`.